test/model/content_test.dart

```diff
@@ -1155,6 +1155,67 @@ class ContentExample {
       ], isHeader: false),
     ]),
   ]);
+
+  static const linkPreviewSmoke = ContentExample(
+    'link preview smoke',
+    'https://pub-14f7b5e1308d42b69c4a46608442a50c.r2.dev/image+title+description.html',
+    '<p><a href="https://pub-14f7b5e1308d42b69c4a46608442a50c.r2.dev/image+title+description.html">https://pub-14f7b5e1308d42b69c4a46608442a50c.r2.dev/image+title+description.html</a></p>\n'
+      '<div class="message_embed">'
```

Comment:
nit: indentation
```suggestion
    '<p><a href="https://pub-14f7b5e1308d42b69c4a46608442a50c.r2.dev/image+title+description.html">https://pub-14f7b5e1308d42b69c4a46608442a50c.r2.dev/image+title+description.html</a></p>\n'
    '<div class="message_embed">'
```

The `div` is a sibling of the `p`, not a child, so the indentation should express that.